### ISSUE TITLE
Corona does not change color on hover

### ISSUE THREAD

**Issue by SebastinSanty** [OSDLabs/WavesWebsite2016#29]
@UtkarshMe : Can you solve the issue?

**Comment by kaivalyar** [comment]
while _hospitality to corona_ was solved, the code was implemented very differently from the other icons on the site, with 2 images being uploaded specifically for this purpose (I'm guessing - correct me if I'm wrong here). It might be simpler to stick to font-awesome. use _fa-glass_ or something. If we really absolutely must have the 2 beer mugs - maybe run all our icon fonts through _icomoon_ or something?

**Comment by SebastinSanty** [comment]
@kaivalyar : Yeah, you are right, two icons were used. Even, I think that the image is an overhead, and moreover not going in with the others. I have not come across icomoon till now, I'll have a look.

BTW @UtkarshMe , Kaivalya has a point, and do you think we need a separate image or can we manage with a font-awesome icon.

PS: Maybe fa-beer will be a better option than fa-glass :P

@UtkarshMe : http://fontawesome.io/icon/beer

**Comment by kaivalyar** [comment]
I had tried both out before commenting itself - glass seems to go better with the site than beer, IMHO. If you want to go with this idea, i'd be happy to make this edit.

**Comment by SebastinSanty** [comment]
I'll take a confirmation from the DoCW side, before.

**Comment by coditva** [comment]
> The beer mugs are what we really want because they represent the fun part of the fest (ie the informals). And we wish to keep it. If it means that the color does not change on hover, it's fine that way. It's not really that big a deal and we can overlook it.
>  -DoCW.

I think it's fine too.

**Comment by kaivalyar** [comment]
Ummm ... DoCW want's to overlook the **performance overhead** too? I mean ... we're going to start uploading images for this?
